web/locales/en/messages.ftl

```diff
@@ -190,6 +197,7 @@ xh = Xhosa
 yi = Yiddish
 yo = Yoruba
 yue = Cantonese
+zgh = Tamazight
```

Comment:
Not sure what causes this and the changes above

Reply:
these are autogenerated (commiting won't hurt/change anything)